server/src/test/java/org/elasticsearch/index/mapper/RootObjectMapperTests.java

```diff
@@ -360,50 +360,150 @@ public void testPassThroughObjectWithAliases() throws IOException {
         assertThat(mapperService.mappingLookup().getMapper("labels.dim"), instanceOf(KeywordFieldMapper.class));
     }
 
+    public void testPassThroughObjectNested() throws IOException {
+        MapperService mapperService = createMapperService(mapping(b -> {
```

Comment:
In the same test case, could you also add a top level attributes passthrough field? Just to check that this doesn’t cause a naming conflict because there’s also an attributes field in the resource object.

Reply:
Done, had a test for that in yaml too.